salt/journal/init.sls

```diff
@@ -6,6 +8,20 @@ maintenance-mode-start:
         - require:
             - nginx-server-service
 
+{% if osrelease not in ["14.04", "16.04"] %}
```

Comment:
probably not at all necessary if the journal instance switches to containers like Chris mentioned